docs/plugins/deploy.md

```diff
@@ -0,0 +1,35 @@
+# Deploy
+
+```
+serverless deploy --stage dev --region us-east-1
+```
+
+Deploys your service to AWS.
```

Comment:
Should those docs be AWS specific or provider independent? I would vote for provider independent. Otherwise I would propose to create a subfolder inside `docs/plugin` which is named after the provider and put provider related docs there (but would also keep the main plugin docs such as `deploy`, `remove` etc.).